Assets/MRTK/SDK/StandardAssets/Controllers/DebugControllers/Models/webxr-models/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please be sure to update the NOTICES.md file to indicate our usage of these models.

Reply:
Done.

Reply:
cgmanifest.json as well, which can be used to generate parts of the Notices file.